index.js

```diff
@@ -2,60 +2,77 @@ var dir = require("node-dir");
 var path = require("path");
 var fs = require("fs");
 
-module.exports = function(folder, cb){
+module.exports = function(folder, cb) {
 
-	dir.files(folder, function(err, files) {
-		if(err){
-			cb(err);
+	var cwd = process.cwd();
+	process.chdir(folder);
```

Comment:
It's just the process. Pretty common for a shell script, which is basically what this is doing. I change it back when I'm done, but even if I didn't, when the process ends, you'd be back in the original directory.